proto/rill/admin/v1/api.proto

```diff
@@ -1066,6 +1066,7 @@ message GetReportMetaRequest {
   string branch = 2;
   string report = 3;
   map<string, string> annotations = 4;
+  string execution_time = 5;
```

Comment:
Can we make this a `google.protobuf.Timestamp` and then move the string formatting into the url builder in `externalURLs.reportOpen`?